<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WidgetEdgeDetection</class>
 <widget class="QWidget" name="WidgetEdgeDetection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item column="1" row="3">
    <widget class="QComboBox" name="cmbOutput">
     <property name="toolTip">
      <string>Output decides what type of information you want from the lines.</string>
     </property>
    </widget>
   </item>
   <item column="1" row="1">
    <widget class="QComboBox" name="cmbType">
     <property name="toolTip">
      <string>The formula decides how the kernel is created, so different formulas choose different values in the kernel, and thus give subtly different results.</string>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </item>
   <item colspan="2" column="0" row="5">
    <widget class="KisDoubleSliderSpinBox" name="sldHorizontalRadius" native="true"/>
   </item>
   <item column="1" row="8">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item colspan="2" column="0" row="7">
    <widget class="QCheckBox" name="chkTransparent">
     <property name="toolTip">
      <string>This will take a desaturated result and use it to determine what is transparent. This is useful for creating edge and fringe effects.</string>
     </property>
     <property name="text">
      <string>Apply result to alpha channel</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight" column="0" row="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Formula:</string>
     </property>
    </widget>
   </item>
   <item colspan="2" column="0" row="6">
    <widget class="KisDoubleSliderSpinBox" name="sldVerticalRadius" native="true"/>
   </item>
   <item alignment="Qt::AlignRight" column="0" row="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Output:</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignVCenter" column="2" row="4" rowspan="3">
    <widget class="KoAspectButton" name="btnAspect" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item column="1" row="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoAspectButton</class>
   <extends>QWidget</extends>
   <header>KoAspectButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>